resources/idlharness.js

```diff
@@ -1310,7 +1321,7 @@ IdlInterface.prototype.get_inheritance_stack = function() {
     while (idl_interface.base) {
         var base = this.array.members[idl_interface.base];
         if (!base) {
-            throw new Error(idl_interface.type + " " + idl_interface.base + " not found (inherited by " + idl_interface.name + ")");
+            throw new IdlHarnessError(`${idl_interface.name} base ${idl_interface.base} not found`);
```

Comment:
I know backtick-strings are nice, but let's keep this file working in older browsers.